### DISCUSSION TITLE
steady state power flows

### DISCUSSION THREAD

**Discussion by Cahui1** [Xyce/Xyce#82] [Q&A]
Hi everyone ，is Newton-Raphson method used for power flow calculation in xyce?

**Comment by Karlsefni2012** [comment]
I assume you are referring to the type of calculations described in this document:

https://xyce.sandia.gov/files/xyce/AppNote-PowerGridModelingWithXyce.pdf

Every type of analysis in Xyce uses Newton's method.    The power-flow calculations described in that document are all either .DC or .TRAN calculations, and they use Newton's method.  This is true of any SPICE-style simulator, BTW.

The only (sort of) exceptions in Xyce to this are small signal analysis methods like .AC and .NOISE.  Both of these methods solve a  linearized small-signal problem during the frequency sweep, which doesn't require Newton's method.  But both .AC and .NOISE are initialized first with a standard DCOP solve, and it uses it.
